addons/debugger/addon.json

```
@@ -1,6 +1,13 @@
 {
   "name": "Debugger",
-  "description": "Adds a new \"debugger\" window to the editor. Allows for logging into the \"Logs\" tab of the debugger window using the \"log\", \"warn\" and \"error\" blocks. The \"breakpoint\" block will pause the project when executed. All running stacks of blocks can be viewed in the \"Threads\" tab of the debugger window, and when paused the \"Step\" button can be used to execute the next block. A graph of frames per second and number of clones can be viewed in the \"Performance\" tab.",
+  "description": "Adds a new \"debugger\" window to the editor. Allows for logging into the \"Logs\" tab of the debugger window using the \"log\", \"warn\" and \"error\" blocks. The \"breakpoint\" block will pause the project when executed. All running stacks of blocks can be viewed in the \"Threads\" tab of the debugger window, and when paused the \"Step\" button can be used to execute the next block. A graph of frames per second and number of clones can be viewed in the \"Performance\" tab. The \"is it in editor?\" block allows to check if the project is running in the editor.",
+  "info": [
+    {
+      "type": "notice",
+      "text": "Debug blocks only work for Scratch Addons users with this enabled. The \"is in editor?\" block cannot therefore it cannot be used as an anti-cheat method.",
```

I don't think mentioning anti-cheat it necessary.
```suggestion
      "text": "Debug blocks, including the \"is in editor?\" block only work for Scratch Addons users with this addon enabled.",
```